packages/css-library/src/utilities/margin.scss

```diff
@@ -0,0 +1,96 @@
+@use 'uswds-core/src/styles/functions' as *;
+@use '../tokens/margin' as *;
+@use './settings' as *;
+
+$u-margin-base: (
+  margin: (
+    base: 'vads-margin',
+    modifiers: null,
+    values: map-collect($tokens-margin),
+    settings: $margin-settings,
+    property: 'margin',
+    type: 'utility',
+  ),
+);
+
+$u-margin-y: (
+  margin-y: (
+    base: 'vads-margin-y',
+    modifiers: null,
+    values: map-collect($tokens-margin-y),
+    settings: $margin-settings,
+    property: (
+      margin-top,
+      margin-bottom,
+    ),
+    type: 'utility',
+  ),
+);
+
+$u-margin-top: (
+  margin-top: (
+    base: 'vads-margin-top',
+    modifiers: null,
+    values: map-collect($tokens-margin-top),
+    settings: $margin-settings,
+    property: 'margin-top',
+    type: 'utility',
+  ),
+);
+
+$u-margin-bottom: (
+  margin-bottom: (
+    base: 'vads-margin-bottom',
+    modifiers: null,
+    values: map-collect($tokens-margin-bottom),
+    settings: $margin-settings,
+    property: 'margin-bottom',
+    type: 'utility',
+  ),
+);
```

Comment:
Could we use the `modifiers` for these like we're doing in the `padding` utilities?

Reply:
We could use it for the utilities that have the same map values, but not for all margin utilities. I'll consolidate the ones we can. 👍